proto/zepben/protobuf/cim/iec61968/infiec61968/infassetinfo/CurrentTransformerInfo.proto

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+syntax = "proto3";
+option java_multiple_files = true;
+option java_package = "com.zepben.protobuf.cim.iec61968.infiec61968.infassetinfo";
+option csharp_namespace = "Zepben.Protobuf.CIM.IEC61968.InfIEC61968.InfAssetInfo";
+package zepben.protobuf.cim.iec61968.infiec61968.infassetinfo;
+
+import "zepben/protobuf/cim/iec61968/assets/AssetInfo.proto";
+import "zepben/protobuf/cim/iec61968/infiec61968/infcommon/Ratio.proto";
+
+/**
+ * Properties of current transformer asset.
+ */
+message CurrentTransformerInfo {
+
+    /**
+     * AssetInfo fields for this CurrentTransformerInfo.
+     */
+    iec61968.assets.AssetInfo ai = 1;
+
+    /**
+     * CT accuracy classification.
+     */
+    string accuracyClass = 2;
+
+    /**
+     * Accuracy limit.
+     */
+    double accuracyLimit = 3;
+
+    /**
+     * Number of cores.
+     */
+    int32 coreCount = 4;
+
+    /**
+     * CT classification; i.e. class 10P.
+     */
+    string ctClass = 5;
+
+    /**
+     * Maximum voltage in volts across the secondary terminals where the CT still displays linear characteristicts.
+     */
+    int32 kneePointVoltage = 6;
+
+    /**
+     * Maximum ratio between the primary and secondary current.
+     */
+    infcommon.Ratio maxRatio = 7;
+
+    /**
+     * Nominal ratio between the primary and secondary current; i.e. 100:5
+     */
+    infcommon.Ratio nominalRatio = 8;
+
+    /**
+     * Ratio for the primary winding tap changer (numerator).
+     */
+    double primaryRatio = 9;
+
+    /**
+     * Rated current on the primary side in amperes.
+     */
+    int32 ratedCurrent = 10;
+
+    /**
+     * Full load secondary (FLS) rating for secondary winding in amperes.
+     */
+    int32 secondaryFlsRating = 11;
+
+    /**
+     * Ratio for the secondary winding tap changer (denominator).
+     */
+    double secondaryRatio = 12;
+
+    /**
+     * Intended usage of the CT; i.e. metering, protection.
+     */
+    string usage = 13;
+
+
```

Comment:
nit: single blank line at end of class preferred

Reply:
strange, IDE must be doing something odd in terms of visualisation as I have an extra line in the IDE in all my files...

Reply:
That is because you did have a blank line at the end of the file, which is not what my nit was about, but the blank line at the end of the class (between last function and close class bracket) where you had two rather than one in this case (three in another, and missing in the third). I have now fixed them and reverted the extra blank line you put at the end of the file.

Reply:
Oh woops, yeah github confused me, thought you meant after the class and github didn't seem to be showing the extra lines.